### PR TITLE
Updated utils package for RHEL 8

### PR DESCRIPTION
**Pull request by rcalixte**
#### Pull Request (PR) description
Updating -utils package for RHEL 8

#### This Pull Request (PR) fixes the following issues
N/A

### CODE REVIEW

**Comment by coreone** [comment]
@alexjfisher is there any timeline on this getting merged to master?  We are actively testing RHEL 8 and this would be extremely useful.

File: manifests/init.pp [hunk 1]
```diff
@@ -126,6 +126,12 @@
     fail("${facts['os']['name']} not supported")
   }
 
+  if (($facts['osfamily' == 'RedHat') and versioncmp($facts['operatingsystemmajrelease'], '7') <= 0) {
```
Comment (bastelfreak) [inline manifests/init.pp]: there is a `]` missing for `$facts[]`?

also both facts are legacy. Please use `os.family` and `os.release.major` instead.

**Comment by rcalixte** [comment]
@bastelfreak This should be good to merge after review. Thank you!

**Comment by alexjfisher** [comment]
> @alexjfisher is there any timeline on this getting merged to master? We are actively testing RHEL 8 and this would be extremely useful.

Sorry I missed this.  I was away at the time.

**Comment by alexjfisher** [comment]
Looks like this PR has been updated to be based on https://github.com/voxpupuli/puppet-yum/pull/143 (which was recently merged)

**Comment by alexjfisher** [comment]
Apparently in EL 8.1 RedHat changed their minds??
See https://github.com/voxpupuli/puppet-yum/pull/159

**Pull request merged**